transaction/core/src/range_proofs/mod.rs

```diff
@@ -1,5 +1,12 @@
 // Copyright (c) 2018-2021 The MobileCoin Foundation
 
+//! Range proofs are used to prove that a set of committed values are all
+//! in a well-defined range, without revealing the values.
+//!
+//! A range proof is relative to a Pedersen Generator. If a prover can construct
+//! a range proof relative to one generator, they cannot construct a range proof
+//! relative to another generator, if those generators are orthogonal.
```

Comment:
@mfaulk this is where i added note about the generator playing the role of the proof of opening, we could expand on this I guess